python/ray/experimental/state/state_cli.py

```diff
@@ -19,6 +22,29 @@
     list_runtime_envs,
 )
 
+AVAILABLE_FORMAT = ["default", "yaml", "json", "table"]
```

Comment:
still prefer enum instead for readability/type checking but unblocking merge, your call...

Reply:
I used enum! Please take a look. I will merge the PR around tomorrow!